lib/sycamore/sycamore/context.py

```diff
@@ -52,6 +52,33 @@ def read(self):
         return DocSetReader(self)
 
 
+def get_val_from_context(
```

Comment:
This function should have its own unit test.

Reply:
good catch I missed